src/app/forgot/student/forgot-student-password-security/forgot-student-password-security.component.spec.ts

```diff
@@ -20,91 +27,131 @@ export class MockStudentService {
+          { provide: ConfigService, useClass: MockConfigService },
+          { provide: RECAPTCHA_V3_SITE_KEY, useValue: '' }
+        ],
+        schemas: [NO_ERRORS_SCHEMA]
+      });
+    })
+  );
 
   beforeEach(() => {
-    TestBed.configureTestingModule({
-      declarations: [ForgotStudentPasswordSecurityComponent],
-      imports: [RouterTestingModule, BrowserAnimationsModule, ReactiveFormsModule],
-      providers: [{ provide: StudentService, useClass: MockStudentService }],
-      schemas: [NO_ERRORS_SCHEMA]
-    });
     fixture = TestBed.createComponent(ForgotStudentPasswordSecurityComponent);
+    recaptchaV3Service = TestBed.inject(ReCaptchaV3Service);
     component = fixture.componentInstance;
+    component.isRecaptchaEnabled = false;
     fixture.detectChanges();
   });
 
-  it('should create', () => {
-    expect(component).toBeTruthy();
-  });
+  changePassword();
+});
 
-  it('should disable the submit button if the answer field is not filled in', () => {
-    fixture.detectChanges();
-    const submitButton = getSubmitButton();
-    expect(submitButton.disabled).toBe(true);
-  });
+async function changePassword() {
+  describe('changePassword()', () => {
+    it('should create', () => {
+      expect(component).toBeTruthy();
+    });
 
-  it('should enable the submit button when the answer field is filled in', () => {
-    component.setControlFieldValue('answer', 'cookies');
-    fixture.detectChanges();
-    const submitButton = getSubmitButton();
-    expect(submitButton.disabled).toBe(false);
-  });
+    it('should disable the submit button if the answer field is not filled in', () => {
+      fixture.detectChanges();
+      const submitButton = getSubmitButton();
+      expect(submitButton.disabled).toBe(true);
+    });
+
+    it('should enable the submit button when the answer field is filled in', () => {
+      component.setControlFieldValue('answer', 'cookies');
+      fixture.detectChanges();
+      const submitButton = getSubmitButton();
+      expect(submitButton.disabled).toBe(false);
+    });
 
-  it('should show the incorrect answer message', () => {
-    submitAndReceiveResponse('checkSecurityAnswer', 'failure', 'incorrectAnswer');
-    expect(getErrorMessage()).toContain('Incorrect answer');
+    it(
+      'should show the incorrect answer message',
+      waitForAsync(() => {
+        submitAndReceiveResponse('checkSecurityAnswer', 'failure', 'incorrectAnswer');
+        expect(getErrorMessage()).toContain('Incorrect answer');
+      })
+    );
+
+    it('should navigate to change password page', () => {
+      const router = TestBed.get(Router);
+      const navigateSpy = spyOn(router, 'navigate');
+      const username = 'SpongebobS0101';
+      const questionKey = 'QUESTION_ONE';
+      const answer = 'cookie';
+      component.username = username;
+      component.questionKey = questionKey;
+      component.isRecaptchaEnabled = false;
```

Comment:
This line `component.isRecaptchaEnabled = false;` can be removed since this is already done in the beforeEach().